docs/ACTION.md

```diff
@@ -127,7 +127,7 @@ done
 
 Note here we are just interested in the payload, but in general you may also want to retrieve other fields.
 
-Note the `actionloop` image will accept any source and will try to run it (if it is possible), while the `action-golang-v1.15`  will instead try to compile the sources assuming it is Golang instead.
+Note the `actionloop` image will accept any source and will try to run it (if it is possible), while the `action-golang-v1.N`  will instead try to compile the sources assuming it is Golang instead.
```

Comment:
got tired of bumping this number (and similar in docs).   I think `v1.N` is clear, but I'm also ok with backing this bit of this change out if someone things it is unclear.

Reply:
I thought this was fine/obvious.